modes/longitudinal/src/toolbarButtons.js

```diff
@@ -631,6 +664,11 @@ const toolbarButtons = [
       ],
     },
   },
+
+  // Register buttons that are used for top level actions
+  StackImageSync,
```

Comment:
It is a bit unexpected to see these here, but I guess they are here so that the tools can be fetched from the ToolbarService and the listeners invoked?